engine/render/src/render/render.cpp

```diff
@@ -953,6 +953,69 @@ namespace dmRender
         return Draw(context, predicate, constant_buffer);
     }
 
+    void DispatchCompute(HRenderContext render_context, uint32_t group_count_x, uint32_t group_count_y, uint32_t group_count_z, HNamedConstantBuffer constant_buffer)
```

Comment:
More or less the same as a draw command, except that there is no render object and world matrices and so on